fileio/private/ft_checkdata.m

```diff
@@ -685,17 +695,17 @@
   data = rmfield(data, 'dof');
 end % if hasdof
 
-if ~isempty(cmbrepresentation)
+if ~isempty(cmbstyle)
```

Comment:
Perhaps I am the only one, but I have quite a few old scripts lying around, where I call ft_checkdata directly, using 'cmbrepresentation' as key. Could we gracefully have ft_checkdata to silently rename 'cmbrepresentation' into 'cmbstyle' for backward compatibility purposes?

Reply:
That is already taken care off at the start of ft_checkdata with 
```
sel = find(strcmp(varargin(1:2:end), 'cmbrepresentation'));
if ~isempty(sel)
  ft_warning('the option ''cmbrepresentation'' is deprecated, please use ''cmbstyle'' instead');
  varargin{2*sel-1} = 'cmbstyle';
end
```

Reply:
ah I missed that, sorry for bringing it up